packages/Algernon.package/ALGRepositoryType.class/class/algertemListWith..st

```diff
@@ -1,13 +1,5 @@
 algertem lists
-algertemListWith: aWorkingCopyLabel
-
-	| repos algertems |
-	algertems := OrderedCollection new.
-	repos := self getReposFor: aWorkingCopyLabel.
-	repos do: [ :each |
-		algertems add: (ALGItem withAlgertype: (self
-			withType: (each description)
-			withPackage: aWorkingCopyLabel))].
-	^ algertems
-	
-
+algertemListWith: aWorkingCopyLabel 
+	^ (self getReposFor: aWorkingCopyLabel)
+		collect: [:each | ALGItem
+				withAlgertype: (self withType: each description withPackage: aWorkingCopyLabel)]
```

Comment:
This might lead to a merge conflict with my `do:` refactorings, but we will care when it comes to that.

Reply:
Alright 👍